deployment/terraform/assets/cluster.tf

```diff
@@ -63,6 +63,8 @@ resource "aws_instance" "app_server" {
   provisioner "remote-exec" {
     inline = [
       "while [ ! -f /var/lib/cloud/instance/boot-finished ]; do echo 'Waiting for cloud-init...'; sleep 1; done",
+      "while ! netcat -z -v security.ubuntu.com 80; do echo 'Waiting for connectivity to security.ubuntu.com...'; sleep 1; done",
```

Comment:
With this polling check in place can we perhaps remove the sleep above?

Reply:
I'll look at this again tomorrow. From Alejandro's output above, this didn't fix things entirely. This does not appear to be the root cause.